src/kata5.reachDestination.js

```diff
@@ -1,5 +1,26 @@
 const reachDestination = (distance, speed) => {
 
+  function reachDestination (distance, speed) {
+
+    let time = Math.floor(distance) / Math.floor(speed);
+
+    let twoDec = time.toFixed(2);
+
+    let twoDecArr = twoDec.toString().split(".");
+
+        if (twoDecArr[1] >= 0 && twoDecArr[1] <= 49) {
+            twoDecArr[1] = 5;
+        } else {
+            twoDecArr[1] = 0;
+      };
+
+    let result = twoDecArr.join('.');
```

Comment:
Use constant variables unless it's your intention to reassign (you should avoid reassignment, but it is acceptable for mathematical operations).